target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -415,6 +415,9 @@ ramips_board_detect() {
 	*"RE350 v1")
 		name="re350-v1"
 		;;
+	*"EA8100")
+		name="linksys,ea8100"
+		;;
```

Comment:
This can be removed, as it will default to DTS compatible.

Reply:
Done! Thank you.